client/components/indicators/index.js

```diff
@@ -19,20 +19,20 @@ const renderLastUpdated = ( lastUpdated ) => {
 	}
 
 	return (
-		<div className="form-setting-explanation indicator__last-updated">
+		<div className="form-setting-explanation indicators__indicator-last-updated">
```

Comment:
No idea why it doesn't throw an error. I've changed this to use the `<FormSettingsExplanation>` component instead of akwardly reusing its CSS classes. In https://github.com/Automattic/woocommerce-services/pull/1051/commits/440da372c3efc3f01b6dcca4d3327c8f833b4651